package.json

```diff
@@ -385,18 +385,12 @@
   },
   "lavamoat": {
     "allowScripts": {
-      "3box>3box-orbitdb-plugins>ipfs-log>orbit-db-identity-provider>orbit-db-keystore>leveldown": false,
```

Comment:
What motivated these changes?

Reply:
hmm i dont remember removing this

Reply:
I see - this change was caused by `@lavamoat/allow-scripts@2.0.1`. Any idea why that patch would have caused these changes?

Reply:
Oh, that patch included the introduction of `@lavamoat/aa`, which totally changed the identifier used for each package. Hmm. Surprising that you'd publish that as a patch, that seems like a breaking change. It causes install to fail unless the config is updated.

Reply:
should have been added in `@lavamoat/allow-scripts@2.0.0`